mix.exs

```diff
@@ -72,6 +72,9 @@ defmodule Livebook.MixProject do
     ]
   end
 
+  defp target_deps(:app), do: [{:app_builder, path: "app_builder"}]
```

Comment:
Once this is an external package, don't we want to track it in `mix.lock`?

Reply:
It will be tracked in the mix.lock too, but it will only be installed if you run under `MIX_TARGET=app`.